bridges/SplCenterBridge.php

```diff
@@ -44,7 +55,7 @@ protected function parseItem(array $item)
     public function getURI()
     {
         if (!is_null($this->getInput('content'))) {
-            return self::URI . '/' . $this->getInput('content');
+            return self::URI . '/resources/' . $this->getInput('content');
```

Comment:
Can you also catch existing feeds that have `news` as `content` value (by rewriting it to `stories`?)